__init__.py

```diff
@@ -1021,6 +1019,8 @@ def export_events(self, output_file, params, format='json', timezone_offset=None
                     current_file = current_file + '.gz'
                 with open(current_file, 'wb') as fp:
                     shutil.copyfileobj(events, fp, buffer_size)
+                if not add_gzip_header and compress:
```

Comment:
not entirely sure what the gzip portion is but I would have assumed that the gzip header would take us here. Confirming that only when there is _no_ header we want to run this `_gzip_file` fn

Reply:
right this is the case where the request was made without the gzip header but they wanted the output compressed to a gzip file.